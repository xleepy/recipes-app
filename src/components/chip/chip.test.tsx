import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import { Chip } from '.';

describe('Chip tests', () => {
  it('should render chip element', () => {
    const { asFragment } = render(<Chip name="lorem" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
