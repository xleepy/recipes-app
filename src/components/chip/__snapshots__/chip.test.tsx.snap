// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Chip tests > should render chip element 1`] = `
<DocumentFragment>
  <div
    class="_chip_fe894d"
  >
    <span
      class="_text_fe894d"
    >
      lorem
    </span>
  </div>
</DocumentFragment>
`;
